arboretum/common/utils.py

```diff
@@ -34,3 +38,25 @@ def parse_seconds(seconds):
             formatted.append(f'{q} {unit}')
         seconds = r
     return ', '.join(formatted)
+
+
+def run_command(cmd, secrets=None):
+    """Run commands in a system."""
+    if type(cmd) == str:
+        cmd = cmd.split(' ')
+    p = subprocess.Popen(
+        cmd,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        universal_newlines=True
+    )
+    stdout, stderr = p.communicate()
+
+    if p.returncode != 0:
+        secrets = secrets or []
+        for s in secrets:
+            cmd = cmd.replace(s, '***')
```

Comment:
I don't think this is right.  After you split it, `cmd` will be a list which does not have a `replace` function.  Did you test this?  There should be a unit test [here](https://github.com/ComplianceAsCode/auditree-arboretum/blob/main/test/test_common_utils.py) for any code that isn't a fetcher or a check like evidence and common utilities.